### PR TITLE
Archive fonds view

### PR DESCRIPTION
**Pull request by justinlevinson**
Created the Archive Fonds view.

- Created a chained GraphQL query (thanks @erchaves) - feel free to steal this for other similar queries where the page requires multiple fetches
- Created the FondsCard to hold content
- Summary cards are a separate Component
- Still missing some data (Archive context, images and type for the cards, some details in the first Details table)
- Applied clipboard-copy and social links styling that somehow got lost in a previous PR

### CODE REVIEW

File: src/Components/FondsCard/index.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    }
+  }
+
+  openModal() {
+    this.props.openModal();
+  }
+
+  showClipboardNotice = () => {
+    this.setState({
+      ...this.state,
```
Comment (erchaves) [inline src/Components/FondsCard/index.js]: See above about redundant ...this.state

File: src/Components/FondsCard/index.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      isClipboardNoticeVisible: true
+    }, () => {
+      setTimeout(this.hideClipboardNotice, 1000)
+    })
+  }
+
+  hideClipboardNotice = () => {
+    this.setState({
+      ...this.state,
```
Comment (erchaves) [inline src/Components/FondsCard/index.js]: See above about redundant ...this.state

File: src/Components/FondsCard/styles.scss [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  }
+  .clipboard-copy-button {
+    border-width: $padding-tiny;
+    padding: $padding-sm;
+    background-color: $color-grey-border-light;
+    display: flex;
+    flex-flow: row;
+    align-items: center;
+    background-color: transparent;
+    background-image: -webkit-gradient(linear, left top, right top, from(rgba(255, 255, 255, 0)), color-stop(75%, #ffffff));
```
Comment (erchaves) [inline src/Components/FondsCard/styles.scss]: Please see note above about similar scss

File: src/Components/FondsCard/styles.scss [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+  }
+
+  // The design calls for these to stretch the full width of the row, without column gutters
+  .archive-items-header .row .col {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  .filter-button {
+    background-color: #FFF;
```
Comment (erchaves) [inline src/Components/FondsCard/styles.scss]: Let's use the $color-white variable

File: src/Components/FondsCard/styles.scss [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.25);
```
Comment (erchaves) [inline src/Components/FondsCard/styles.scss]: It looks like this is used many times. Let's make a new var for it in /src/shared-styles/_variables.scss

File: src/Components/SearchSingleArchive/styles.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
+@import '../../shared-styles/variables';
+@import '../../shared-styles/mixins';
+@import '../../shared-styles/materialize/materialize-variables.scss';
+
+.component-single-object {
+  .btn{
+      border: 2px solid blue;
```
Comment (erchaves) [inline src/Components/SearchSingleArchive/styles.scss]: This looks repeated as well if we want to make a $var for it.

File: src/gqlQueries/archive.js [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+  }
+  objectName
+  objectStatus
+  collectionType
+  collectionCode
+`;
+
+// Todo, can we fix the api so that we can re-use the fragment instead like we could before?
+// I think this should be (plural) objects($id: Int!)
+const gqlQueries = {
```
Comment (erchaves) [inline src/gqlQueries/archive.js]: Let's clean up this copied code if we can by removing things that don't make sense here like the "byMedium" bit for example. Or at least add a TODO indicating that it's a temp copy from objects that should be cleaned up.

File: src/Components/SearchSingleArchive/styles.scss (hunk 6 above)
Comment (justinlevinson) [reply src/Components/SearchSingleArchive/styles.scss]: I think there's a lot, and it's only going to grow - I found myself copy-pasting a lot and started to get concerned. I put in a TODO and will do a big cleanup tomorrow AM.

**Pull request merged**